tests/integration/policy/test_policy_pack/index.ts

```diff
@@ -16,6 +16,15 @@ if (!packName) {
                 name: "s3-no-public-read",
                 description: "Prohibits setting the publicRead or publicReadWrite permission on AWS S3 buckets.",
                 enforcementLevel: "mandatory",
+                configSchema: {
+                    properties: {
+                        message: {
+                            "type": "string",
```

Comment:
nit: I think type, minLength and maxLength do not need to be quotes & may make this look cleaner.